docs/about.rst

```diff
@@ -58,13 +58,17 @@ We think that's a pretty fair deal, and we're big believers in open source. If y
 Presentations
 -------------
 
-Yellowbrick has enjoyed the spotlight at a few conferences and in several presentations. We hope that these videos, talks, and slides will help you understand Yellowbrick a bit better.
+Yellowbrick has enjoyed the spotlight in several presentations at recent conferences. We hope that these notebooks, talks, and slides will help you understand Yellowbrick a bit better.
```

Comment:
Slight wording change to add in "notebooks." Also, our list of conference presentations is growing so in the first sentence I replaced "few" with "recent".